### PR TITLE
s/verison/version/

### PR DESCRIPTION
**Pull request by ezyang**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #118749

Signed-off-by: Edward Z. Yang <ezyang@meta.com>

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @chenyang78 @aakhundov @kadeng

Differential Revision: [D53287845](https://our.internmc.facebook.com/intern/diff/D53287845)

### CODE REVIEW

**Review by malfet — APPROVED** [review]
I like "verison" more, but sure :)

**Review by albanD — APPROVED** [review]
SGTM if you're happy with the bc-breaking change

File: torch/fx/experimental/_config.py [hunk 1]
```diff
@@ -35,7 +35,7 @@
 inject_EVALUATE_EXPR_flip_equality_TESTING_ONLY = False
 
 # [@compile_ignored: debug] Validate that ShapeEnv's version key is updated correctly
-validate_shape_env_verison_key = False
+validate_shape_env_version_key = False
```
[the remark this answers is not in the document]
Comment (ezyang) [reply torch/fx/experimental/_config.py:38]: 👹

**Comment by angelayi** [comment]
@angelayi has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D53287845).

**Pull request closed without merging**